arangod/Aql/SortNode.cpp

```diff
@@ -159,10 +158,10 @@ SortInformation SortNode::getSortInformation(
         result.isValid = false;
         return result;
       }
-      result.criteria.emplace_back(std::make_tuple(
-          const_cast<ExecutionNode const*>(setter),
-          std::string(buffer->c_str(), buffer->length()), (*it).ascending));
-      buffer->reset();
+      result.criteria.emplace_back(
+          std::make_tuple(const_cast<ExecutionNode const*>(setter),
+                          std::move(buffer), (*it).ascending));
+      buffer.clear();
```

Comment:
seems to be redundant given `std::move(buffer)` above

Reply:
Also changed.